docs/developers/get_started/getting_started_linux_cmake.md

```diff
@@ -121,6 +121,7 @@ Translate a source MLIR into an IREE module:
 ```shell
 # Assuming in IREE source root
 $ ../iree-build/iree/tools/iree-translate \
+    -iree-input-type=mhlo \
```

Comment:
We could add a new canonical test .mlir file that doesn't contain mhlo instead of adding this flag to developer docs all over.
(the user-facing docs will _definitely_ not want to specify this flag)

Reply:
For more context: we're moving to make the native input to the compiler be a somewhat standardized set of linalg+flow, rather than a mix of mhlo/linalg/flow/tosa/etc. depending on where the program came from. Anyone going through the primary user-facing tools will be working without special flags. Within the project we'll still have tests that use other dialects (at least in the short term), so tests and developer guides may need to use the flag.

Reply:
That makes sense. However, there's still a user-facing doc of `docs/website/docs/building-from-source/android.md` that still requires this flag. Most of these getting started docs point the mlir file to [iree/tool/test/iree-run-module.mlir](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-module.mlir). If we can update that file (or at least for the user-facing example) it would be great.

Reply:
Let's switch to this MLIR then:

```mlir
func @abs(%input : tensor<f32>) -> (tensor<f32>) attributes { iree.module.export } {
  %result = absf %input : tensor<f32>
  return %result : tensor<f32>
}
```
and update any `i32`s to `f32`s in the docs + test comments